xtb/qcschema/harness.py

```diff
@@ -137,9 +144,13 @@ def run_qcschema(
                 atomic_input.keywords["electronic_temperature"]
             )
 
-        # We want the full printout from xtb
-        calc.set_verbosity(VERBOSITY_FULL)
-        calc.set_output(fd.name)
+        # Work out how verbose the printing from xtb should be
+        if isinstance(verbosity, str):
+            verbosity = _verbosity_flags[verbosity.lower()]
```

Comment:
I think case sensitive input is fine, also getting from the dict with fallback seems like a cleaner choice.
```suggestion
        verbosity = _verbosity_flags.get(verbosity, verbosity)
```